frontends/web/src/components/backups/backups.css

```diff
@@ -59,14 +59,23 @@
     display: flex;
     flex-wrap: wrap;
     flex-direction: row-reverse;
-    gap: var(--space-half);
     margin-top: var(--space-default);
 }
 
+@media (min-width: 769px) {
+    .backupButtons > * + * {
```

Comment:
what does `* + *` mean?

Reply:
:)  it's "the owl" selector 🦉 https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/ 

basically targets anything that has is not first child, can change to `*:not(:first-child)` if you want